code/modules/client/preference/link_processing.dm

```diff
@@ -700,6 +700,16 @@
 					if(new_relation)
 						active_character.nanotrasen_relation = new_relation
 
+				if("physique")
+					var/new_physique = input(user, "Choose your descriptor for how built your character is on glance.", "Character Preference") as null|anything in list("rail thin", "thin", "average build", "well-built", "muscular", "overweight")
```

Comment:
I'd suggest pulling these out to a list stored somewhere, possibly as a GLOB. These feel a little too important to only be defined here.